src/components/gltf-model-plus.js

```diff
@@ -449,6 +468,38 @@ class GLTFHubsPlugin {
         }
 
         node.extras.gltfIndex = i;
+
+        if (qsTruthy("newLoader")) {
+          /**
+           * This guarantees that components that add Object3Ds (ie. through addObject3DComponent) are not attached to non-Object3D
+           * entities as it's not supported in the BitECS loader.
+           * This was supported by the AFrame loader so this extension ensures backwards compatibility with all the existing scenes.
+           * For more context about this see: https://github.com/mozilla/hubs/pull/6121
+           */
+          if (node.mesh !== undefined || node.camera !== undefined) {
```

Comment:
We may need to also check [glTF punctual lights extension](https://github.com/KhronosGroup/glTF/blob/main/extensions/2.0/Khronos/KHR_lights_punctual/README.md)? Three.js glTF loader built-in plugin creates Three.js `Light` object inheriting `Object3D`.